entrypoint/createcluster/createcluster_suite_test.go

```diff
@@ -20,7 +20,7 @@ func TestMain(m *testing.M) {
 	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
 	flag.Parse()
 
-	cfg, err = config.AddConfigEnv("../../config")
+	cfg, err = config.AddConfigEnv("../../config/.env")
```

Comment:
Question: Is there a way to get rid of this `../../` and have relative or absolute path derived? I remember there was a BasePath but not sure if that can be used here...

Reply:
Also in some places I see we are using `./config/.env` as well. So if we can have the `.env` path saved in shared, I think that will be great

Reply:
sure i can create some function do handle.
base path will not be enough